_sass/components/_map.scss

```diff
@@ -76,21 +76,21 @@
     color: $dark-gray;
     display: inline-block;
     height: 55px;
-    padding: 33px 0px 10px 0px;
     margin-left: 10px;
+    padding: 33px 0 10px 0;
```

Comment:
Shorthand form for property `padding` should be written more concisely as `33px 0 10px` instead of `33px 0 10px 0`